server/src-lib/Hasura/GraphQL/Execute/Action.hs

```diff
@@ -82,19 +81,19 @@ data ActionWebhookErrorResponse
 $(J.deriveJSON (J.aesonDrop 5 J.snakeCase) ''ActionWebhookErrorResponse)
 
 data ActionWebhookResponse
-  = AWRArray ![J.Object]
-  | AWRObject !J.Object
+  = AWRArray ![Map.HashMap G.Name J.Value]
+  | AWRObject !(Map.HashMap G.Name J.Value)
```

Comment:
As we've found out, not all JSON objects can be represented in GraphQL; more specifically, object key names are unrestricted strings in JSON while GraphQL Name needs to match `^[_a-zA-Z][_a-zA-Z0-9]*$`. Are we sure this change is fine?